docs/README.md

```diff
@@ -1,13 +1,25 @@
 # Documentation For Developers
 
-Here are some documents important for TEAMMATES developers.
+Welcome to the documentation of the TEAMMATES project.
+
+Although it may look intimidating, the documentation will make working on the project much easier. Read through them well and it'll make contributing to TEAMMATES that much more enjoyable!
```

Comment:
Should we say "Although it may look intimidating"? That sounds a big negative. Agreed with Samson on the second part.